makefile.cargo

```diff
@@ -20,8 +20,7 @@ ifneq (,$(CARGO_FEATURE_DEBUGMOZJS))
 	CFLAGS += -g -O0 -DDEBUG -D_DEBUG
 endif
 
-MOZJS_OUTDIR = $(shell find $(OUT_DIR)/../.. -name 'mozjs_sys-*' -type d)/out
-CFLAGS += -I$(MOZJS_OUTDIR)/dist/include
+CFLAGS += -I$(DEP_MOZJS_OUTDIR)/dist/include
```

Comment:
Sorry, you’re right. `/out` is indeed included in the environment variable. I restarted the Travis build and it’s green now.